projectfiles/CodeBlocks/README.md

```diff
@@ -19,6 +19,26 @@
     NOTE: You must make sure to download the 32-bit version.
     Building a 64-bit Wesnoth executable on Windows is currently not supported and will
     fail with the SDK package provided in the next step.
+	
+	To resolve out of memory errors when creating debug builds, follow these steps:
+	
+	1. If you use 32-bit Windows, run Admin command prompt
+	```
+	bcdedit/set IncreaseUserVa 3072
+	```
+	
+	2. Install MASM32;
+	
+	3. Open `cmd`( as Admin too);
```

Comment:
```suggestion
	3. Open `cmd` (also as admin);
```

Reply:
Will powershell do or must it be MS-DOS "cmd"?

Reply:
@GregoryLundberg PowerShell would indeed work here, but I think it's best to keep the instructions simple and only talk about CMD.